api/src/main/kotlin/team/backend/infrastructure/quiz/repository/QuizReaderImpl.kt

```diff
@@ -19,4 +22,41 @@ class QuizReaderImpl(
             where(col(Quiz::id).equal(id))
         }
     }
+
+    override suspend fun getIds(): List<Long> {
+        return queryFactory.listQuery {
+            selectMulti(col(Quiz::id))
+            from(entity(Quiz::class))
+        }
+    }
+
+    override suspend fun getQuizIdsThatCanBeSolvedByMember(job: Job, tier: Tier): List<Long> {
```

Comment:
I refactor it. 

get is Correct. Take a look at the internal implementation of listQuery().

Reply:
@BAEKJungHo 
Can you tell me what part of it you think is right for `get`?

Reply:
@mangchhe 
If you look inside `listQuery()`, you use `resultList()`, but if the value is empty, it returns EmptyList instead of returning an error.

If it is found, it should be taken as optional, and if it is null, the service should handle the appropriate exception.

On the other hand, get is far from the idea of optional.

Reply:
@mangchhe 

__find examples__

```kotlin
override suspend fun find(id: Long): Member? {
    return queryFactory.singleQueryOrNull{
        select(entity(Member::class))
        from(entity(Member::class))
        where(col(Member::id).equal(id))
    }
}
```
```kotlin
val member = memberReader.find(authorization.toLong()) ?: throw NotFoundMemberException()
```

Reply:
@BAEKJungHo 
I think there's always a value when we use `get`, So no value should allow null and throw exceptions.
But `find` doesn't always have a value. It could be empty.
If we use it as get, it seems to mean that there is a value in the list.

Reply:
@mangchhe I refactor it. 

Looking at JPA's naming rules, I think empty is also a rule of find.